Makefile

```diff
@@ -0,0 +1,113 @@
+SHELL=bash
+SOURCE_DIR = $(shell pwd)
+BIN_DIR = ${SOURCE_DIR}/bin
+COMPOSER = composer
+CI_DIR = ${BIN_DIR}/tool-php-ci
+AWS_SHARED_CREDENTIALS_FILE ?= $(HOME)/.aws/credentials
+
+define printSection
+	@printf "\033[36m\n==================================================\n\033[0m"
+	@printf "\033[36m $1 \033[0m"
+	@printf "\033[36m\n==================================================\n\033[0m"
+endef
+define replaceDotenv
+	test -f .env.dev.local || cp .env.dev .env.dev.local
+	TEMPFILE="$(shell mktemp)" && \
+	sed -e 's@^$1=.*@$1=$2@' .env.dev.local > "$$TEMPFILE" && \
+	mv "$$TEMPFILE" .env.dev.local
+endef
+
+.PHONY: all
+all: install quality test test-dependencies
+
+.PHONY: ci
+ci: quality test test-dependencies
```

Comment:
since the continous integration platform of this project does not use `make ci`, we should either:
- update the CI to effectively call `make ci`
- simply remove this target

Reply:
update the CI to effectively call make ci ✅